xml/Microsoft.Build.Tasks/DownloadFile.xml

```diff
@@ -171,8 +167,7 @@
       <Parameters />
       <Docs>
         <summary>This method is called by the build engine to begin task execution. A task uses the return value to indicate whether it was successful. If a task throws an exception out of this method, the engine will automatically assume that the task has failed.</summary>
-        <returns>
-          <see langword="true" />, if successful; <see langword="false" /> otherwise.</returns>
+        <returns><see langword="true" />, if successful; <see langword="false" /> otherwise.</returns>
```

Comment:
```suggestion
        <returns>
          <see langword="true" />, if successful; <see langword="false" /> otherwise.</returns>
```

Reply:
When the comment starts with an HTML tag, it's best to put it on a newline. Otherwise the next CI build will do it for you.